app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express") ;
+const app = express();
+const { getTopics } = require("./controllers/topics.controllers.js");
+
+app.use(express.json());
```

Comment:
you don't need express.json() til you are doing a post, or patch (ie. there is a request body). please remove. 